scripts/health_check.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+
+import argparse
+import json
+from enum import Enum
+
+import requests
+
+
+ENV_ENUM = Enum(
+    'ENV',
+    [
+        ('dev', 'https://addons-dev.allizom.org'),
+        ('stage', 'https://addons.allizom.org'),
+        ('prod', 'https://addons.mozilla.org'),
+        # TODO: maybe we could use the local environmnet here
+        ('test', ''),
+    ],
+)
+
+
+class Fetcher:
+    def __init__(self, env: ENV_ENUM, verbose: bool = False):
+        self.environment = ENV_ENUM[env]
+        self.verbose = verbose
+
+    def _fetch(self, path: str) -> dict[str, str] | None:
+        url = f'{self.environment.value}/{path}'
+        if self.verbose:
+            print(f'Requesting {url} for {self.environment.name}')
+
+        try:
+            response = requests.get(url)
+            response.raise_for_status()
```

Comment:
We return a `500` error in the monitor view if one or more services is down alongside a valid json response, so if you raise for status, `data` is not set and that causes your script to raise `UnboundLocalError`. We want to do the `data = response.json()` bit first because it can work even if the response status is not 20x.

Reply:
So  really a non 200 response is kind of not important for this we could just skip it entirely right? We will already catch the json error if we don't get a good response or how would you suggest I proceed?

Reply:
Haven't tried it but we should still account for the possibility of a 500 coming from an unhandled exception somewhere in the view, so just have to be careful about that - we wouldn't return a json or if somehow we are, we wouldn't have the `state` key that we check for, so maybe that's ok as long as it fails later in `main()`.

Reply:
So to clarify, should I check for status at all? I guess we should rely on the response being valid json and then inspect the values instead of relying on the http status code. wdyt?

Reply:
There is still a possibility to raise a 500 from an unhandled exception in the view while still returning json but that's something we currently only do for API exceptions, so I guess it's fine to ignore status for now, and only rely on the response being valid json (if it's not valid json, then you have an unhandled 500 on your hands)